R/mod_comments_table.R

```diff
@@ -0,0 +1,58 @@
+# Module UI
+  
+#' @title   mod_comments_table_ui and mod_comments_table_server
+#' @description  A shiny Module.
+#'
+#' @param id shiny id
+#' @param input internal
+#' @param output internal
+#' @param session internal
+#'
+#' @rdname mod_comments_table
+#'
+#' @keywords internal
+#' @export 
+#' @importFrom shiny NS tagList 
+#' @importFrom shinydashboard box
+#' @importFrom shinycssloaders withSpinner
+mod_comments_table_ui <- function(id){
+  ns <- NS(id)
+  tagList(
+    box(
+      withSpinner(
+        DT::dataTableOutput(ns("comments_table"))
+      ),
+      title = span(tagList(icon("comments"), "Comments")),
+      solidHeader = TRUE,
+      collapsible = TRUE,
+      status = "primary",
+      width = NULL
+    )
+  )
+}
+    
+# Module Server
+    
+#' @rdname mod_comments_table
+#' @export
+#' @keywords internal
+#' 
+#' @import dplyr
+#' @import stringr
+    
+mod_comments_table_server <- function(input, output, session, comments_promise) {
+  ns <- session$ns
```

Comment:
unused variable

Reply:
As far as I understand this is needed for the server in order to put UI elements in the proper namespace ;) https://shiny.rstudio.com/articles/modules.html